src/applications/enrollment-verification/components/EnrollmentVerificationAlert.jsx

```diff
@@ -18,9 +18,12 @@ const successAlert = submissionResult => (
 );
 const warningAlert = (
   <va-alert status="warning" visible>
-    <h3 slot="headline">
+    <p
+      className="vads-u-font-size--h3 vads-u-font-weight--bold"
+      slot="headline"
+    >
       We’re missing one or more of your enrollment verifications
-    </h3>
+    </p>
```

Comment:
Is there a heading that would be appropriate to use in the hierarchy/not cause a 508 compliance issue? I think an `<h2>`, `<h4>`, or `<h5>` would be more appropriate than a paragraph tag, semantically, as these are headings.